src/storage/src/controller.rs

```diff
@@ -143,10 +181,19 @@ pub trait StorageController: Debug + Send {
         collections: Vec<(GlobalId, CollectionDescription<Self::Timestamp>)>,
     ) -> Result<(), StorageError>;
 
+    /// Acquire an immutable reference to the export state, should it exist.
```

Comment:
This naming is throwing me for a loop, now we have sinks and exports and sources and ingest descriptions and whatnot... :sweat_smile:

Reply:
Here's a perfect issue for you! https://github.com/MaterializeInc/materialize/issues/13782 :)